packages/blueprints/launch-blueprint/src/blueprint.ts

```diff
@@ -52,8 +52,14 @@ export interface Options extends ParentOptions {
 
   /**
    * @readOnly
+   * @deprecated use `paremeters` property instead
    */
   options: Tuple<[string, string]>[];
+
+  /**
+   * @readOnly
+   */
+  parameters: DynamicKVInput[];
```

Comment:
should this be optional?

Reply:
Default specified in the defaults instead.